site/sitecontent/terms.md

```diff
@@ -0,0 +1,183 @@
+
+### Identity Theft and Privacy
+
+Users that misleadingly appropriate the identity of another person are not permitted. Users may not post other people's personally identifying or confidential information, including but not limited to credit card numbers, Social Security Numbers, and driver's and other licence numbers. You may not post information such as other people's passwords, usernames, phone numbers, addresses and e-mail addresses unless already publicly accessible on the Web.
+
+### Hate Content, Defamation, and Libel
+
+Hate speech and other objectionable content that is unlawful, defamatory, and fraudulent. Note that an allegation of defamatory expression, in and of itself, does not establish defamation. The truth or falsehood of a bit of expression is a key element in establishing defamation, and we are not in a position to make that sort of fact-based judgement. That said, if we have reason to believe that a particular statement is defamatory (a court order, for example), we will remove that statement.
+
+### Sexually Explicit Content
+
+Accounts that use to host sexually explicit or pornographic material will be suspended.
+
+### Disruptions and Exploits
+
+We will terminate accounts and block addresses of anyone who attempts unauthorized use of LibreCores.
+
+### Mass Registration and Automation
+
+Accounts registered automatically or systematically will be permanently suspended.
+Suspended Accounts
+
+If you are found to be in violation of any of the above policies, you will receive a notice via email. Unless you explain or correct your behaviour within 72 hours, your account will be suspended. We will do our best to work with you and ensure a fair outcome in all cases. We reserve the right to immediately suspend, without notice, any content, account, or IP address which we determine to be systematically generated, spam, or potentially damaging to our service or infrastructure. Public access to the content of a suspended account will be blocked, and the LibreCores pages replaced with a generic page, or redirected to the LibreCores front page. The ability to add/edit content will also be disabled.
+
+## Privacy Policy
+
+This privacy policy sets out how the Free and Open Source Silicon Foundation CiC ("FOSSi Foundation") uses and protects any information that you give the Free and Open Source Silicon Foundation CiC when using LibreCores (the "Website").
+
+FOSSi Foundation is committed to ensuring that your privacy is protected. Should we ask you to provide certain information by which you can be identified when using this website, then you can be assured that it will only be used in accordance with this privacy statement.
+
+FOSSi Foundation may change this policy from time to time by updating this page. You should check this page from time to time to ensure that you are happy with any changes. This policy is effective from 1st April 2017.
+
+### What we collect
+
+We may collect the following information:
```

Comment:
How about all the other information people enter when creating projects?

Reply:
Right, will extend the text